examples/gym/train_ppo_batch_gym.py

```diff
@@ -164,14 +110,49 @@ def make_batch_env(test):
     obs_normalizer = chainerrl.links.EmpiricalNormalization(
         obs_space.low.size, clip_threshold=5)
 
+    winit_last = chainer.initializers.LeCunNormal(1e-2)
+
     # Switch policy types accordingly to action space types
-    if args.arch == 'FFSoftmax':
-        model = A3CFFSoftmax(obs_space.low.size, action_space.n)
-    elif args.arch == 'FFMellowmax':
-        model = A3CFFMellowmax(obs_space.low.size, action_space.n)
```

Comment:
I approve to remove mellowmax from the example.

Reply:
Yes, I'm aware that this PR removes mellowmax model. I think nobody uses it.